cupy/core/_kernel.pxd

```diff
@@ -93,7 +93,7 @@ cdef class _Ops:
     cdef _Ops from_tuples(object ops, routine)
 
     # Queries a single op from input arguments.
-    cdef _Op guess_routine(
+    cpdef _Op guess_routine(
```

Comment:
How about adding a note that this function is used from `fusion.py`, so as not to forget reverting this to `cdef` when `fusion.py` is cythonized?